engines/scumm/actor.h

```diff
@@ -70,11 +66,15 @@ struct CostumeData {
 	uint32 heCondMaskTable[16];
 
 	void reset() {
+		animCounter = 0;
+		soundCounter = 0;
+		soundPos = 0;
```

Comment:
This is a small behavior change. Looks harmless to me, but someone who knows these classes better than me should approve.

Reply:
If these 3 vars don't belong there, please move them back to the constructor. No need for experiments here :-)